ms/MSOper/MSMetaData.cc

```diff
@@ -3399,6 +3399,8 @@ void MSMetaData::_computeScanAndSubScanProperties(
     uInt chunkSize = nrows/nchunks;
     if (nrows % nchunks > 0) {
         ++chunkSize;
+        // integer division
+        nchunks = nrows/chunkSize + 1;
     }
```

Comment:
isn't the maximum size `nchunks + 1` for when nrows is not divisible by nchunks?

    fut[nchunks + 1] or fut[nchunks + (nrows & nchunks == 0 ? 0 : 1)]